source/content/mysql-slow-log.md

```diff
@@ -87,23 +157,26 @@ $ vi site_name_slow_1s_noDupes.txt
 # User@Host: pantheon[pantheon] @  [10.223.192.139]  
 # User@Host: pantheon[pantheon] @  [10.223.192.68]  
 # User@Host: pantheon[pantheon] @  [10.223.192.87]  
-SET timestamp=1418627746;SELECT node.title AS node_title, node.nid AS nid, node_counter.totalcount AS node_counter_totalcount, ga_stats_count_pageviews_today.count AS ga_stats_count_pageviews_today_countFROM node nodeLEFT JOIN node_counter node_counter ON node.nid = node_counter.nidLEFT OUTER JOIN ga_stats_count ga_stats_count_pageviews_today ON node.nid = ga_stats_count_pageviews_today.nid AND (ga_stats_count_pageviews_today.metric='pageviews' AND ga_stats_count_pageviews_today.timeframe='today') WHERE (( (node.status = '1') AND (node.type IN  ('story')) )) ORDER BY ga_stats_count_pageviews_today_count DESC LIMIT 5 OFFSET 0;  
+SET timestamp=1418627746;SELECT node.title AS node_title, node.nid AS nid, node_counter.totalcount AS node_counter_totalcount, ga_stats_count_pageviews_today.count AS ga_stats_count_pageviews_today_countFROM node nodeLEFT JOIN node_counter node_counter ON node.nid = node_counter.nidLEFT OUTER JOIN ga_stats_count ga_stats_count_pageviews_today ON node.nid = ga_stats_count_pageviews_today.nid AND (ga_stats_count_pageviews_today.metric='pageviews' AND ga_stats_count_pageviews_today.timeframe='today') WHERE (( (node.status = '1') AND (node.type IN  ('story')) )) ORDER BY ga_stats_count_pageviews_today_count DESC LIMIT 5 OFFSET 0; 
 ```
-This particular query is, at it's worst, examining 132,363 records to return 5, while taking a full second to do so. That would make it a fairly good candidate for refactoring, since most sites prefer queries to execute in milliseconds.
+
+This particular query is, at its worst, examining 132,363 records to return 5, while taking a full second to do so. That would make it a fairly good candidate for refactoring, since most sites prefer queries to execute in milliseconds.
 
 ## Look at the slow queries by hour
 
-Another method is to look at slow queries by the hour to see if there are spikes in slow queries that correspond to site traffic patterns.
+Another method is to look at slow queries by the hour to see if there are spikes in slow queries that correspond to site traffic patterns:
 
-    grep Time  endpointas90kkud28a236-slow.log | cut -d: -f1,2 | sort | uniq -c  
+```bash
+grep Time  endpointas90kkud28a236-slow.log | cut -d: -f1,2 | sort | uniq -c  
 
-    70 # Time: 140708 10  
-    71 # Time: 140708 11  
-    49 # Time: 140708 12  
-    77 # Time: 140708 13  
-    77 # Time: 140708 14  
-    35 # Time: 140708 15  
-    76 # Time: 140708 16  
+70 # Time: 140708 10  
+71 # Time: 140708 11  
+49 # Time: 140708 12  
+77 # Time: 140708 13  
+77 # Time: 140708 14  
+35 # Time: 140708 15  
+76 # Time: 140708 16  
+```
 
 This means there were 70 slow queries between 10 and 11AM. That is roughly even distribution, which probably means there are a few slow queries that keep repeating.
```

Comment:
@sarahg is the time reference based on the timezone on the pantheon server (UTC), the user's local system, or something else?